src/scss/tom-select.bootstrap4.scss

```diff
@@ -197,22 +195,22 @@ $select-arrow-offset: calc(#{$select-padding-x} + 5px) !default;
 	height: auto;
 	border: none;
 	background: none;
-	//box-shadow: none;
+
+	// box-shadow: none;
```

Comment:
Linting is adding a space before comments, maybe these comments are not even necessary here ...
```suggestion
```

Reply:
Probably not, but I didn't write the original code so maybe it was a reminder for something, I'll remove those comments.